### PR TITLE
Auth token

### PR DESCRIPTION
**Pull request by CarlKarlQarl**

### CODE REVIEW

File: src/pages/LoginNSignup/LoginNSignup.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import React, {useState}from 'react';
-import {Login} from '../../api/api';
+import React, { useState } from 'react';
+import axios from "axios"
```
Comment (tylercodes1) [inline src/pages/LoginNSignup/LoginNSignup.js:2]: Not urgent, but we will factor all HTTPRequests to the api folder.

**Comment by CarlKarlQarl** [comment]
Not the cleanest implementation, but definitely functional. I'd like to review it with someone, and find a way to have Redux check for the token in storage when the app starts, and have the token get set to the Redux's store when it is created.

File: src/pages/LoginNSignup/LoginNSignup.js [hunk 2]
```diff
@@ -29,15 +37,49 @@ function LoginNSignup() {
             [e.target.name]: e.target.value
         });
     }
+
+    const handleLogin = (event) => {
+        event.preventDefault()
+        axios.post("http://localhost:5000/api/auth/signin", {
```
Comment (tylercodes1) [inline src/pages/LoginNSignup/LoginNSignup.js:43]: Didn't know you could do api calls synchronously XD

**Review by tylercodes1 — APPROVED** [review]
Made some general comments, but looks good. Some try catch blocks could be put in a .catch() ajax method call, but it's not urgent.  Would love a demonstration of accessing local storage tomorrow!

**Pull request merged**